src/libcore/iter/adapters/chain.rs

```diff
@@ -207,6 +207,29 @@ impl<A, B> DoubleEndedIterator for Chain<A, B> where
         }
     }
 
+    #[inline]
+    default fn nth_back(&mut self, mut n: usize) -> Option<A::Item> {
+        match self.state {
+            ChainState::Both | ChainState::Back => {
```

Comment:
Since you are editing this, can you please change it to `if let ChainState::Both | ChainState::Back = self.state { ... }`?

Reply:
Done